camelot/art/awesome/extract.py

```diff
@@ -0,0 +1,14 @@
+import json
```

Comment:
kan je dit gebeuren hier uitleggen ?

Reply:
Het extract.py script dient om de originele font awesome metadata (camelot/art/awesome/icons.json, 3.1M) om te zetten naar camelot/art/awesome/name_to_code.json (31K). Dit laatste bestand w gebruikt om font awesome icon namen te mappen naar een unicode code.

Ik heb dit bestand toegoevoegd om dit process eenvoudiger te maken in de toekomst (bijv. nieuwe font awesome release). Het is mischien beter om dit te verplaatsen naar het camelot fabfily.py als "fontawesome_update" commando?

Reply:
en waar komt die icons.json vandaan ?

idd : 

- verplaats het script uit de camelot code, maar steek het idd in de fab of zo
- verplaats ook de 3M icons.json, ofwel uit de code, ofwel afhalen vanwaar die komt ?  zodat die niet steeds mee gepackaged w

Reply:
zet er aub ook wat uitleg bij ...

Reply:
Dat icons.json bestand zit in het fontawesome zip bestand als je een release download.

Ik heb dit aangepast, zie nieuwe push.